Makefile

```diff
@@ -11,6 +11,12 @@ all:
 	dune build @install
 	@test -L bin || ln -s _build/install/default/bin .
 
+# Build only scilla-checker and scilla-runner
+blockchain:
```

Comment:
```suggestion
slim:
```

A better name for this target would be `slim` since it's quite common that both `full` and `slim` images are provided. I will push another `Dockerfile.slim` to this PR in order to use this target. I will also make the change to the auto-build on docker hub. As a result, when this PR is merged, there will be two tags there distinguishing these two types of image:

- `zilliqa/scilla:latest`: the latest version will full build
- `zilliqa/scilla:slim`: the latest version with slim build